testnet/stacks-node/src/neon_node.rs

```diff
@@ -1475,11 +1493,22 @@ impl InitializedNeonNode {
 
         if let Some(ref snapshot) = &self.last_burn_block {
             debug!(
-                "Tenure: Notify sortition! Last snapshot is {}/{} ({})",
+                "Tenure: Notify sortition! Last snapshot is {}/{} ({}) sortition {}",
                 &snapshot.consensus_hash,
                 &snapshot.burn_header_hash,
-                &snapshot.winning_stacks_block_hash
+                &snapshot.winning_stacks_block_hash,
+                &snapshot.sortition_id
```

Comment:
This would be much more readable as a k-v pattern.

Reply:
Yup; fixed